host_core/lib/host_core/web_assembly/imports.ex

```diff
@@ -253,9 +253,34 @@ defmodule HostCore.WebAssembly.Imports do
     %{token | authorized: true}
   end
 
+  # Deny invocation due to missing link definition for a contract ID and link name
+  defp invoke(
+         _token = %{
+           verified: false,
+           authorized: false,
```

Comment:
Nit: do we need this `authorized` check? I noticed `authorized` will always be false if `verified` is false

Reply:
Fair, we will deny it if the link isn't verified